frontend/src/components/AdaptableButton.tsx

```diff
@@ -0,0 +1,38 @@
+import dashCircle from '../../images/dashCircle.svg'
+import { useState } from 'react';
+
+interface AdaptableButtonProps {
+    label: string;
+    disabled?: boolean;
+    showDash?: boolean;
+}
+
+const AdaptableButton: React.FC<AdaptableButtonProps> = ({ 
+    label,
+    disabled = false,
+    showDash = false,
+ }) => {
+    const [isClicked, setIsClicked] = useState(false);
+
+    const handleClick = () => {
+        if (!disabled && !showDash) {
+            setIsClicked(prevState => !prevState);
+        }   
+    };
+
+    const hoverEffects = !showDash && !disabled ? 'hover:bg-secondary-bg-500 hover:text-white' : '';
```

Comment:
personally like the hover effects but pls remove since we're developing for mobile browsers which don't respond well to hovers (user is clicking with fingers)